lib/polyn/event.rb

```diff
@@ -93,23 +92,27 @@ def to_h
         "time"            => time,
         "data"            => Utils::Hash.deep_stringify_keys(data),
         "datacontenttype" => datacontenttype,
-        "polyntrace"      => Utils::Hash.deep_stringify_keys(polyntrace),
         "polyndata"       => Utils::Hash.deep_stringify_keys(polyndata),
       }
     end
 
     ##
     # Get the Event `source` prefixed with reverse domain name
     def self.full_source(source = nil)
-      root = Polyn.configuration.source_root
-      name = Polyn::Naming.dot_to_colon("#{domain}:#{root}")
+      root    = Polyn.configuration.source_root
+      parts   = [domain, root]
+      combine = lambda do |items|
```

Comment:
Huh. I thought Rubocop yelled at us when we used `lambda do` instead of `-> (items) {`

Reply:
It yelled if I did a single line with `lambda`, but if it was multi-line it insisted I use `lambda`

Reply:
Ugh. Thanks, Rubocop, for your helpful consistency. 🙄